src/content/tutorials/deploy-a-react-app-with-create-react-app/index.md

```diff
@@ -0,0 +1,114 @@
+
+```toml
+---
+filename: wrangler.toml
+highlight: [3, 4, 5]
+---
+# ... previous wrangler config
+
+[site]
+bucket = './build'
+entry-point = 'workers-site'
+```
+
+With a configured `wrangler.toml` file, it’s time to build your project, and publish it to Workers. Run `npm run build` to tell `create-react-app` to build your site, and `wrangler publish` to deploy it to Workers:
+
+```sh
+---
+header: Build and publish your application
+---
+$ npm run build
+$ wrangler publish
+```
+
+After deploying your project, open up your browser to see your static site in action!
+
+![Result](./media/demo.png)
+
+## How it works
+
+The Workers Site feature is designed to work with as little as configuration as possible: since the process for deploying static sites is fairly consistent, regardless of framework or language, you shouldn’t need to spend a lot of time configuring your project or writing any additional code to serve your site on Workers.
+
+That being said, if you’re interested in how Workers serves static sites, this section will lightly document how the underlying script works, and what it does each time a user makes a request to your site.
+
+Like all Workers scripts, the static site feature begins by listening for incoming `fetch` events to your application — these are incoming requests from a client, such as a browser or a phone:
+
+```js
+---
+filename: index.js
+---
+addEventListener('fetch', event => {
+  event.respondWith(handleRequest(event.request))
+})
+```
+
+When the script receives an incoming [`Request`](https://developer.mozilla.org/en-US/docs/Web/API/Request), it looks at the `pathname`, such as `/workers`, and looks up an associated file uploaded to Workers KV. If that file is found, a new [`Response`](https://developer.mozilla.org/en-US/docs/Web/API/Response) is generated, with a matching [MIME type](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types) in the `Content-Type` header of the response — for instance, if the path `/workers.jpg` is requested, a new response is returned with the header `Content-type: image/jpg`.
+
+After fetching assets from [Workers KV](https://developers.cloudflare.com/workers/reference/storage), the static site template will cache them in Cloudflare’s powerful CDN. When subsequent users request `/index.html`, Cloudflare’s CDN will transparently serve a cached version.
+
+## Resources
+
+In this tutorial, you built and published a static site to Workers. If you’d like to see the full source code for this application, visit the [repo on GitHub](https://github.com/signalnerve/react-workers-template).
+
+If you want to get started building your own projects, check out the quick-start templates we’ve provided in our [Template Gallery](/templates).
```

Comment:
Update to new ending seen in other new l18n tutorial (Template Gallery): 

https://github.com/cloudflare/workers-docs-engine/blob/5b0395e72f3c4d7d052fda7a4d8fc4d1b3946490/src/content/tutorials/localize-a-website/index.md#L281